packages/components/src/List/Toolbar/SelectSortBy/SelectSortBy.component.js

```diff
@@ -75,15 +86,17 @@ function SelectSortBy({ field, id, isDescending, onChange, options, t }) {
 				</NavDropdown>
 			)}
 			{selected && (
-				<NavItem
-					id={id && `${id}-order`}
-					onClick={onChangeOrder}
-					aria-label={t('LIST_CHANGE_SORT_BY_ORDER', {
-						defaultValue: 'Change sort order. Current order: {{sortOrder}}.',
-						sortOrder: currentSortOrderLabel,
-					})}
-				>
-					{currentSortOrderLabel}
+				<NavItem>
+					<OrderChooser
+						id={id ? `${id}-order` : 'tc-list-toolbar-sort-order'}
```

Comment:
Don't set a default id if no `props.id`. Or if you need one, generate it. Having the same id for 2 lists in the page is ko.